### DISCUSSION TITLE
Exclude lines

### DISCUSSION THREAD

**Discussion by SeanPTOConnor** [WinMerge/winmerge#1383] [Ideas]
Would it be possible to add a feature where the user could enter a list of keywords, and when one of those keywords appears in a line those lines are not compared?

**Comment by sdottaka** [comment]
It may be a bit different from what you expected, but WinMerge has a similar feature called Line filters.

![image](https://user-images.githubusercontent.com/98126/176182250-8446988a-f97c-4796-9e3f-b75df57a9422.png)
![image](https://user-images.githubusercontent.com/98126/176182599-7b215ed8-509f-4d3b-b0ce-d5c6d772e2dc.png)


For information about Line filters, see the following URL: 

https://manual.winmerge.org/en/Filters.html#Filters_LineFilterSyntax
